hyperon_das/api.py

```diff
@@ -26,44 +31,78 @@
     QueryAnswer,
     QueryOutputFormat,
     QueryParameters,
+    config,
 )
 
 
 class DistributedAtomSpace:
     def __init__(
-        self,
-        database: DatabaseType,
-        host: Optional[str] = None,
-        port: Optional[str] = None,
+        self, das_type: DasType = DasType.CLIENT.value, **kwargs
     ) -> None:
-        self._db_type = database
+        self._type = das_type
+        self.__remote_das = []
```

Comment:
Why using two "_" as prefix? Does it have a special meaning?

Reply:
Because I did want to put it as a private attribute